### PR TITLE
enpoind relacionado con el front signuoForm PARA REGISTRAR USUARIO RE…

### PR DESCRIPTION
**Pull request by adilsoN353**
Changes were made to front folders in signupForm and in addition to that, create the link path with the enpoint in the operational Signup flux, users can now register in our database

### CODE REVIEW

**Review by greyzeids — APPROVED** [review]
ahora si

File: migrations/env.py
Comment (Lichatti) [inline migrations/env.py:1]: folder migrations should not be updated

File: .gitignore
Comment (Lichatti) [inline .gitignore:1]: this change has been added to another Pull Request

**Review by Lichatti — CHANGES_REQUESTED** [review]
check .gitignore and migrations/env.py
I have doubts about the Pipfile file

**Review by adilsoN353 — COMMENTED** [review]
carpetas eliminadas para realizar PR

**Review by Lichatti — APPROVED** [review]

**Pull request closed without merging**